### ISSUE TITLE
Problem with the attachments storage after upgrading to https

### ISSUE THREAD

**Issue by tillchen** [4minitz/4minitz#487]
Hi! The 4minitz works perfectly on our university server. Thank you for that:)

But there's one problem: before upgrading the site to https, I was able to find the attachments in the 4minitz_storage/attachments folder; however after using https, though I can still upload/download attachments, the attachments folder is always empty. So could you help with the issue?

Thank you a lot!

**Comment by derwok** [comment]
Thanks for the kind words!

htps/https and empty upload folder should not be corellated if upload/download work in general.

First check: have you changed the 
`"ROOT_URL": "http://www.your-domain.com:3100/",` to replace http with https in settings.json?

Second check: 
Whenever you upload a file, the 4minitz server log shows where the file was stored - with absolute path... have a look and find out. Probably its a different folder than you think?

**Comment by tillchen** [comment]
Hi again:) Thank you for your reply. I did change the root_url to https. And after checking the log, it showed that the file is stored in /4minitz_bin/bundle/programs/server/4minitz_storage/attachments/. But there doesn't seem to be a directory called /4minitz_bin/ on my server. So could it because  I changed the url from jacobs.university:3100 to minitz.jacobs.university? Thanks! @derwok

**Comment by derwok** [comment]
Please attach your settings.json (without passwords) to this issue.

**Comment by derwok** [comment]
One more hint: the log you see is from the viewpoint of the server running inside the docker image. So the path/4minitz_bin/bundle/programs/server/ is the server path inside the docker image... This is **bad** news for you, because it seems your attachments are stored INDISDE the docker image. So, when you stop your docker image - they are gone!

But I wonder why the docker path is prepended to the 4minitz_storage/attachments?
The default setting for the attachments of the docker image is:
`"storagePath": "/4minitz_storage/attachments"` - maybe you accidently removed the initilal "slash" `/`? The slash is important!

**Comment by tillchen** [comment]
[4minitz_settings copy.txt](https://github.com/4minitz/4minitz/files/2975605/4minitz_settings.copy.txt)
So here is the settings file:) @derwok

**Comment by tillchen** [comment]
Ah, So I just have to add the back slash in the settings? Let me try. Thanks! @derwok

**Comment by derwok** [comment]
No backslash! A forward slash, please! 
The initial docker setup had it there - I think you removed it one day...
This is not a good idea...  :-)

**Comment by derwok** [comment]
The same problem here:
`"targetDocPath": "4minitz_storage/protocols",`

**Comment by tillchen** [comment]
Oh, sorry I meant forward slash when I typed back slash😅. (I sometimes confuse these two.) And I added the forward slashes to both paths. Problem solved! Thank you so much!!! 

(BTW, I think I changed these before because I was referring to the settings_sample.json file. And in that file, the forward slash is missing before 4minitz_storage. So maybe it's better to change them in the sample file as well?)

Anyway, thank you so much:) @derwok
